dashboard/app/views/levels/editors/fields/_blockly.html.haml

```diff
@@ -1,6 +1,6 @@
 - content_for(:head) do
   - if @level.is_a?(Blockly) && !@level.uses_droplet?
-    - if @level.is_a?(Dancelab) || @level.is_a?(Poetry)
+    - if [Bounce, Dancelab, Flappy, GamelabJr, Music].any? { |level_type| @level.is_a?(level_type) }
```

Comment:
Outside the scope of this PR: Do we have multiple sources of truth in our codebase for the list of fully migrated labs? If so, how can we consolidate so that when we migrate a lab, the list of level edit pages/block pools/etc. (making up things I think might fall under this umbrella) automatically get updated without us having to remember all the spots to update?

Reply:
Oh could we use this from each level model?

```
  def uses_google_blockly?
    true
  end
```

Reply:
I'm on another branch now, but I wonder if this would work:

```
if @level.is_a?(Blockly) && @level.uses_google_blockly?
```

Reply:
Confirmed that ^ works! FYI @bencodeorg @ebeastlake

Reply:
Nice! Good callout Emily